src/main/groovy/io/seqera/wave/service/inspect/ContainerInspectService.groovy

```diff
@@ -73,5 +73,5 @@ interface ContainerInspectService {
      * @return
      *      The {@link ContainerSpec} object modelling the container image inspect metadata
      */
-    ContainerSpec containerSpec(String containerImage, PlatformId identity)
+    ContainerSpec containerSpec(String containerImage, PlatformId identity, String platform)
```

Comment:
Let's use `String containerImage, String platform, PlatformId identity` whenever possible

Reply:
done

Reply:
Let's do also [here](https://github.com/seqeralabs/wave/pull/660/files#diff-16868f80a923c2d26d795773f1c8fc23cd1af435c9a469cc27bcee7f08186eb1R99) for consistency

Reply:
done